src/app/design/styles/color/index.html

```diff
@@ -369,7 +369,7 @@
     <div class="definition-props">
       <div class="definition-usage">
         <div class="sky-field-label">Usage</div>
-        <div>Background for categorized containers or for containers that have no categories.</div>
+        <div>Background for categorized content.</div>
```

Comment:
Just curious, did we decide that this is NOT the background color for empty containers?

Reply:
Is removing the "no categories" part from the original version correct?

Reply:
I believe it is. I think the scenario where it applies is too specific and edge-case-y to worry about in the description at this atomic level.

Reply:
Sounds good. Thanks. Just wanted to make sure.